checklists/alz_checklist.en.json

```diff
@@ -530,7 +531,7 @@
         {
             "category": "Resource Organization",
             "subcategory": "Subscriptions",
-            "text": "If AD on Windows Server, establish a dedicated identity subscription in the Indentity management group, to host Windows Server Active Directory domain controllers",
+            "text": "If Entra Domain Services on Windows server, establish a dedicated identity subscription in the Identity management group, to host the domain controllers",
```

Comment:
Question on this - I am not aware that Windows Server Active Directory is being changed to Entra Domain Services.  The product documentation still refers to it as AD.  Should we keep the old name here?

Reply:
For this section, could we change this to:

If servers will be used for Identity services, like domain controllers, establish a dedicated identity subscription in the Identity management group to host the services.

Reply:
Good spotting here, I'll update based on recent comment